### PR TITLE
Zoma/checksum

### PR DESCRIPTION
**Pull request by coldplaying42**
Merge chrimc/arrays into zoma/integrating. 

Add checksum for integrate work item. That is, if the hash of the old file does not equal to that of the new file, show the message that the user should manually merge this file. 

Keep integrate as an independent command. And --integrate could also be added into dialog:generate.

### CODE REVIEW

File: packages/chatdown/package.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
     "rimraf": "^2.6.3",
     "ts-node": "^8.1.0",
     "tslint": "^5.16.0",
-    "typescript": "^3.5.3"
+    "typescript": "3.5.3"
   },
```
Comment (chrimc62) [inline packages/chatdown/package.json]: did you mean to change all of the non-dialog files?

File: packages/chatdown/utils/index.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ const fs = require('fs-extra');
 const path = require('path');
 const crypto = require('crypto');
 const mime = require('mime-types');
-const { ActivityTypes, AttachmentLayoutTypes } = require('botframework-schema');
+const { ActivityTypes, AttachmentLayoutTypes } = require('@chimc62/botframework-schema');
```
Comment (chrimc62) [inline packages/chatdown/utils/index.js]: >chimc62 [](start = 59, length = 7)

Why is this needed?

File: packages/dialog/bad-schema.form.dialog [hunk 3]
```diff
@@ -1 +0,0 @@
-undefined
```
Comment (chrimc62) [inline packages/dialog/bad-schema.form.dialog]: Why did you remove this file?  It was in the tests.

**Comment by cleemullins** [comment]
This PR is 457 files, has conflicts, and is effectively not reviewable. Closing. 

If we need this, please refactor and resubmit.
